svc-hl7-validation/src/main/kotlin/gov/cdc/dex/validation/service/ValidationController.kt

```diff
@@ -45,36 +47,296 @@ class ValidationController(@Client("redactor") redactorClient: HttpClient, @Clie
         this.structureClient = structureClient
     }
 
-    @Get(value = "/heartbeat")
+    @Get(value = "/heartbeat", produces = [MediaType.TEXT_PLAIN])
+    @Operation(summary="Used by application startup process. Returns 'hello' if all is well.")
```

Comment:
more important than "hello", is that it returns a HTTP status of 200 if all is well!